src/posit/connect/external/databricks.py

```diff
@@ -234,105 +225,64 @@ def __init__(self, token_endpoint_url: str, client_id: str, client_secret: str):
-    """`CredentialsStrategy` implementation which supports interacting with Service Account OAuth integrations on Connect.
-
-    This strategy callable class returns a `PositContentCredentialsProvider` when hosted on Connect, and
-    its `local_strategy` strategy otherwise.
-
-    Examples
-    --------
-    NOTE: in the example below, the `PositContentCredentialsStrategy` can be initialized anywhere that
-    the Python process can read environment variables.
-
-    ```python
-    from posit.connect.external.databricks import PositContentCredentialsStrategy
-
-    import pandas as pd
-    from databricks import sql
-    from databricks.sdk.core import ApiClient, Config, databricks_cli
-    from databricks.sdk.service.iam import CurrentUserAPI
+class PositWorkbenchCredentialsStrategy(CredentialsStrategyWrapper):
+    """
+    """
+    def __init__(self, config: Config):
+        self.config = config
 
-    DATABRICKS_HOST = "<REDACTED>"
-    DATABRICKS_HOST_URL = f"https://{DATABRICKS_HOST}"
-    SQL_HTTP_PATH = "<REDACTED>"
+    def auth_type(self) -> str:
+        return POSIT_WORKBENCH_AUTH_TYPE
 
-    # NOTE: currently the databricks_cli local strategy only supports auth code OAuth flows.
-    # https://github.com/databricks/cli/issues/1939
-    #
-    # This means that the databricks_cli supports local development using the developer's
-    # databricks credentials, but not the credentials for a service principal.
-    # To fallback to service principal credentials in local development, use
-    # `PositLocalContentCredentialsStrategy` as a drop-in replacement.
-    posit_strategy = PositContentCredentialsStrategy(local_strategy=databricks_cli)
+    def __call__(self, *args, **kwargs) -> CredentialsProvider: # noqa: ARG002
+        if self.config.token is None:
+            raise ValueError("Missing value for field 'token' in Config.")
 
-    cfg = Config(host=DATABRICKS_HOST_URL, credentials_strategy=posit_strategy)
+        def cp():
+            return {"Authorization": f"Bearer {self.config.token}"}
+        return cp
 
-    databricks_user_info = CurrentUserAPI(ApiClient(cfg)).me()
-    print(f"Hello, {databricks_user_info.display_name}!")
 
-    query = "SELECT * FROM samples.nyctaxi.trips LIMIT 10;"
-    with sql.connect(
-        server_hostname=DATABRICKS_HOST,
-        http_path=SQL_HTTP_PATH,
-        credentials_provider=posit_strategy.sql_credentials_provider(cfg),
-    ) as connection:
-        with connection.cursor() as cursor:
-            cursor.execute(query)
-            rows = cursor.fetchall()
-            print(pd.DataFrame([row.asDict() for row in rows]))
-    ```
+class PositConnectCredentialsStrategy(CredentialsStrategyWrapper):
     """
-
-    def __init__(
-        self,
-        local_strategy: CredentialsStrategy,
-        client: Optional[Client] = None,
+    """
+    def __init__(self,
+                 client: Optional[Client] = None,
+                 user_session_token: Optional[str] = None,
     ):
```

Comment:
```suggestion
    """
    def __init__(self,
                 client: Optional[Client] = None,
                 *,
                 user_session_token: Optional[str] = None,
                 content_session_token: Optional[str] = None,
    ):
```

Would keep these explicitly keyword arguments and then allows the dev to pass the token with an appropriately named param.